code_climate.rb

```diff
@@ -0,0 +1,161 @@
+require "net/http"
+require "json"
+require "active_support/core_ext"
+
+DEFAULT_CODECLIMATE_URL = "https://api.codeclimate.com"
+
+class CodeClimateRequest
+  def initialize
+    @base_uri  = ENV["CODECLIMATE_URL"] || DEFAULT_CODECLIMATE_URL
```

Comment:
We may want to use the config gem.  I think some of the other files in this repo started using it.

Reply:
Nevermind, this is for later.  I don't see it in the Gemfile.  I guess scripts are using custom code :disappointed: